anxcloud/testutils/environment/naming.go

```diff
@@ -0,0 +1,178 @@
+package environment
+
+import (
+	"fmt"
+	"math/rand"
+)
+
+var (
+	adjectives = [...]string{
```

Comment:
huh, what's that `[...]string` syntax for? Why not `[]string`?

Reply:
[...]type creates an array of a fixed size (so just a short notation of [62]string. You dont have to count by yourself)instead of a slice that also contains length capacity and the pointer to an underlying array. I just thought since i dont plan to ever append to this list at runtime an array is sufficient. 
But to be fair I don't care i could also make it a slice

Reply:
its gone